### PR TITLE
Replace evil library

### PR DESCRIPTION
**Pull request by dollerbill**
Also update ruby to 2.7.2

👤

### CODE REVIEW

**Comment by dollerbill** [comment]
I didn't bump the version in the workflow, I'll let you make that decision 🎩

**Comment by thorncp** [comment]
Since http is a gem, would you mind adding a Gemfile and .lock that includes it?

**Comment by thorncp** [comment]
I just did some work in master to set up Bundler, using the inline feature (which I learned existed today): https://github.com/thorncp/lgtm/commit/f9ccc270829870247ef113c6203bd296edbb03d6. We should add http there

File: build.rb [hunk 1]
```diff
@@ -1,19 +1,19 @@
 require "fileutils"
-require "net/http"
 require "json"
 
 require "bundler/inline"
 
 gemfile do
   source "https://rubygems.org"
+  gem 'http', '4.4.1'
```
Comment (thorncp) [inline build.rb]: Would you mind changing these to double quotes for consistency?

**Review by thorncp — COMMENTED** [review]
Please add a random emoji to the end of the subject line of the commit message :small_blue_diamond:

**Review by thorncp — APPROVED** [review]

**Pull request merged**